plotting/track.py

```diff
@@ -0,0 +1,441 @@
+import datetime
+import time
+
+import dateutil.parser
+import matplotlib.gridspec as gridspec
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+import pytz
+from geopy.distance import distance
+import visvalingamwyatt as vw
+from flask_babel import gettext
+from scipy.interpolate import interp1d
+
+import plotting.utils as utils
+import plotting.colormap as colormap
+from data import open_dataset
+from data.utils import datetime_to_timestamp
+from data.observational import db, Platform, DataType, Station, Sample
+from data.observational.queries import get_platform_variable_track
+from plotting.plotter import Plotter
+
+
+class TrackPlotter(Plotter):
+
+    def __init__(self, dataset_name: str, query: str, **kwargs):
+        self.plottype: str = "track"
+        super(TrackPlotter, self).__init__(dataset_name, query, **kwargs)
```

Comment:
Python 3 doesn't need the arguments in `super` anymore. They're implicit. `super().__init__(dataset_name, query, **kwargs)`. This applies to the other instances of this below.